<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="youtube_video_url_lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="video_only_pushButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>140</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Video Only</string>
    </property>
   </widget>
   <widget class="QPushButton" name="audio_only_pushButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>170</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Audio Only</string>
    </property>
   </widget>
   <widget class="QPushButton" name="video_and_audio_pushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>451</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>VIDEO+AUDIO DOWNLOAD</string>
    </property>
   </widget>
   <widget class="QComboBox" name="video_options_comboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>471</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="audio_options_comboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>471</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="youtube_video_url_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Youtube Video url</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>video options</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>audio options</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="output_path_lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="output_path_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Downloads to...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_pushButton">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="load_pushButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>LOAD</string>
    </property>
   </widget>
   <widget class="QComboBox" name="progressive_options_comboBox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>611</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="progressive_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>progressive AV options (720p or lower)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="progressive_download_pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Prog Download</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filename_lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="filename_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Filename</string>
    </property>
   </widget>
   <widget class="QPushButton" name="status_pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>STATUS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
